SocialCareCaseViewerApi.Tests/V1/IntegrationTests/IntegrationTestSetup.cs

```diff
@@ -70,5 +66,14 @@ public void AfterAllTests()
         {
             _connection.Dispose();
         }
+
+        private void WipeDatabase()
+        {
+            DatabaseContext.Database.ExecuteSqlRaw("DROP SCHEMA dbo CASCADE");
+            var filePath = TestContext.CurrentContext.WorkDirectory.Replace(Path.Combine("SocialCareCaseViewerApi.Tests", "bin", "Debug", "netcoreapp3.1"), "");
```

Comment:
@LBHTKarki I couldn't think of a better way to load in the db schema file. Do you have any ideas?